### DISCUSSION TITLE
Cannot use the namespace.  e.g.  using json = nlohmann::json;

### DISCUSSION THREAD

**Discussion by sgeva** [nlohmann/json#4063] [Q&A] [answered]
I am including json.hpp in my project, using Xcode.  This seems to be done correctly since the header file is found.

But I cannot declare this:
using json = nlohmann::json;

The error I get:
Use of undeclared identifier 'nlohmann';

![image](https://github.com/nlohmann/json/assets/11728135/e7356e56-3336-43f7-844d-ab66a90bb4d8)

I thought the json.hpp was all that was needed, but perhaps there is more I need to do.
Any advice will be appreciated.  Thanks.

**Comment by sgeva** [comment]
Had to use the single include file and then  it worked.  Not quite satisfactory, but works.

**Comment by gregmarr** [comment, marked as answer]
If you are using the multiple-file version, you need to include `nlohmann/json.hpp` and add the directory that contains `nlohmann` to your include path.  You can't put `nlohmann` itself into your include path, as the headers include each other using the `nlohmann/something.hpp` or `nlohmann/detail/something.hpp` path form.
